### PR TITLE
calibre: cosmetical modifications

### PR DESCRIPTION
**Pull request by AndersonTorres**
To make @r-rmcgibbo happy!

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113149` at 4d7b43a3 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/86bd73120b2386faa96b7845617cce31">calibre</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

`patches` should be a list.
Near pkgs/applications/misc/calibre/default.nix:36:3:
```
   |
36 |   patches = [
   |   ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113149` at 4d7b43a3 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/faa33c460187cfc4eb6c8936bdbe6e85">calibre</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

`patches` should be a list.
Near pkgs/applications/misc/calibre/default.nix:36:3:
```
   |
36 |   patches = [
   |   ^
```
</li>
  </ul>
</details>

**Comment by AndersonTorres** [comment]
@rmcgibbo I think this is a bug in the bot!

**Comment by rmcgibbo** [comment]
hehe. yes, this is a bug (https://github.com/jtojnar/nixpkgs-hammering/issues/23). sorry :(...

**Comment by rmcgibbo** [comment]
To be clear, it's a bug code that I wrote and slipped past Jan into hammering. All the blame is on me.

File: pkgs/applications/misc/calibre/default.nix [hunk 1]
```diff
@@ -59,8 +57,7 @@ mkDerivation rec {
       setup/build.py
 
     # Remove unneeded files and libs
-    rm -rf resources/calibre-portable.* \
-           src/odf
+    rm -rf src/odf resources/calibre-portable.*
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/calibre/default.nix:60]:
```suggestion
    rm -r src/odf resources/calibre-portable.*
```

**Pull request merged**